dapr/proto/operator/v1/operator.proto

```diff
@@ -17,7 +17,7 @@ package dapr.proto.operator.v1;
 
 import "google/protobuf/empty.proto";
 
-option go_package = "github.com/dapr/dapr/pkg/proto/operator/v1;operator";
+option go_package = "github.com/dapr/dapr/dapr/proto/operator/v1;operator";
```

Comment:
It was in the `dapr/pkg/proto` directory before, and now it is unified under the dapr/dapr` independent module.

Reply:
So that  we can use `github.com/dapr/dapr/dapr@commit-id` to be referenced by go-sdk.

![image](https://user-images.githubusercontent.com/20457624/180125920-bb0760cd-2942-44ab-9a82-1afb6b6c39a5.png)